BenchPress/Helpers/Azure/ResourceGroup.psm1

```diff
@@ -1,24 +1,49 @@
+<#
+.SYNOPSIS
+  Helper function for Resource Group
+
+.DESCRIPTION
+  Helper function for Resource Group
+
+.PARAMETER ResourceGroupName
+  The name of the resource group
+
+.EXAMPLE
+  Get-ResourceGroup -ResourceGroupName "rgbenchpresstest"
+#>
 function Get-ResourceGroup {
   [CmdletBinding()]
   param (
     [Parameter(Mandatory=$true)]
-    [string]$resourceGroupName
+    [string]$ResourceGroupName
   )
 
-  $resource = Get-AzResourceGroup $resourceGroupName
+  $resource = Get-AzResourceGroup $ResourceGroupName
   return $resource
 }
 
-function Get-ResourceGroupExists {
+<#
+.SYNOPSIS
+  Helper function for Resource Group
+
+.DESCRIPTION
+  Helper function for Resource Group
+
+.PARAMETER ResourceGroupName
+  The name of the resource group
+
+.EXAMPLE
+  Get-ResourceGroup-ResourceGroupName "rgbenchpresstest"
```

Comment:
`ResourceGroup` -> `ResourceGroupExist` and missing space before `-ResourceGroupName`